test/transferFrom.js

```diff
@@ -3,13 +3,35 @@ const InkProtocol = artifacts.require("./mocks/InkProtocolMock.sol")
 
 contract("InkProtocol", (accounts) => {
   let protocol
+  let sender
+  let recipient
+  let agent
+  let amount
 
   beforeEach(async () => {
     protocol = await InkProtocol.new()
+    sender = accounts[1]
+    recipient = accounts[2]
+    agent = accounts[3]
```

Comment:
`agent` does not apply to all tests, let's move this to just that one test that it's used in.